### PR TITLE
SLING-10929 : Wrap the exceptions of onFile an provide a path to the output

### PR DESCRIPTION
**Pull request by niekraaijmakers**
https://issues.apache.org/jira/browse/SLING-10929

From : 

Screenshot 2021-11-18 at 11 53 02

To:

Screenshot 2021-11-18 at 11 51 39
:

### CODE REVIEW

File: src/main/java/org/apache/sling/feature/cpconverter/ContentPackage2FeatureModelConverter.java [hunk 1]
```diff
@@ -496,7 +496,13 @@ private void process(@NotNull String entryPath, @NotNull Archive archive, @Nulla
 
     @Override
     protected void onFile(@NotNull String entryPath, @NotNull Archive archive, @NotNull Entry entry) throws IOException, ConverterException {
-        process(entryPath, archive, entry);
+        try {
+            process(entryPath, archive, entry);
+        }catch (ConverterException ex){
```
Comment (anchela) [inline src/main/java/org/apache/sling/feature/cpconverter/ContentPackage2FeatureModelConverter.java:501]: formatting: please add a whitespace between } and catch and between ) and {

File: src/main/java/org/apache/sling/feature/cpconverter/ContentPackage2FeatureModelConverter.java [hunk 2]
```diff
@@ -496,7 +496,13 @@ private void process(@NotNull String entryPath, @NotNull Archive archive, @Nulla
...
+            throw new ConverterException("ConverterException occured on path " + entryPath + " with message: " + ex.getMessage(), ex);
+        }catch(IOException ex){
```
Comment (anchela) [inline src/main/java/org/apache/sling/feature/cpconverter/ContentPackage2FeatureModelConverter.java:503]: same here: please stick to the formatting the project which is whitespace between } and catch and between ) and {

**Review by anchela — CHANGES_REQUESTED** [review]
@niekraaijmakers just minor formatting findings.
also the PR needs to reference the JIRA ticket in the message

**Review by anchela — APPROVED** [review]
hi @niekraaijmakers looks good to me

**Comment by karlpauls** [comment]
Looks good to me as well - I merged it. Thanks!

**Pull request merged**